extras/data/POS_F2C/c.yaml

```diff
@@ -0,0 +1,2 @@
+NCPositivityCharge: all
```

Comment:
the values "up", "down", "strange", and "all" are consistent with apfel: https://github.com/scarrazza/apfel/blob/4426f689a95fbde94012083dd55a9af0edcc5928/src/DIS/SelectCharge.f https://github.com/scarrazza/apfel/blob/0ec3ca854e625c791066918cea2a7fc05ec59c99/src/DIS/initParametersDIS.f#L53 - I guess on our side we could do charm ... but as you know, for charm it is much more complicated since we're playing on kernels instead ...

Reply:
You're right: https://github.com/scarrazza/apfel/search?q=SelectedCharge
Perhaps `strange` is also missing.

Ok, let's leave it like this, and we might reconsider this in #152